src/Microsoft.AspNetCore.DataProtection/ActivatorExtensions.cs

```diff
@@ -1,91 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Reflection;
-using Microsoft.AspNetCore.Cryptography;
-using Microsoft.AspNetCore.DataProtection.Internal;
-using Microsoft.Extensions.DependencyInjection;
-
-namespace Microsoft.AspNetCore.DataProtection
-{
-    /// <summary>
-    /// Extension methods for working with <see cref="IActivator"/>.
-    /// </summary>
-    internal static class ActivatorExtensions
```

Comment:
I wasn't around for the resolution of this - are we confident that we can/should remove this feature?